R/validation.R

```diff
@@ -0,0 +1,43 @@
+validate_date <- function(date) {
```

Comment:
Worth adding a little extra description on this function? I know it's not exported but will be helpful for future maintainers

Reply:
Done